odoo/addons/base/models/ir_actions_report.py

```diff
@@ -107,7 +107,7 @@ class IrActionsReport(models.Model):
     print_report_name = fields.Char('Printed Report Name', translate=True,
                                     help="This is the filename of the report going to download. Keep empty to not change the report filename. You can use a python expression with the 'object' and 'time' variables.")
     attachment_use = fields.Boolean(string='Reload from Attachment',
-                                    help='If you check this, then the second time the user prints with same attachment name, it returns the previous report.')
+                                    help='If set to true, then the second time the user prints with same attachment name, it returns the previous report.')
```

Comment:
Wording: I feel like not everybody might know off the cuff that a checkbox represents a boolean which has true/false as value, so maybe `If enabled,` or `If checked,` might sound comprehensible to more people.
```suggestion
                                    help='If enabled, then the second time the user prints with same attachment name, it returns the previous report.')
```

Reply:
Well, that is pure technical field so I dont think this is needed but I will Improve it on next commit